### PR TITLE
Feat/mappings

### PR DESCRIPTION
**Pull request by anngvu**
Create directory with some mappings to cBioPortal for data that we have. Additional context:

- If other clinical data are obtained as described [in this ticket](https://sagebionetworks.jira.com/browse/NFINT-721?atlOrigin=eyJpIjoiY2IzOTI5YjNiYjExNGQ4YzhiMDQzZjBiYTA5NTIyYjAiLCJwIjoiaiJ9), the mappings will be expanded in a separate PR. 
- See the README.
- [This R code](https://github.com/nf-osi/nfportalutils/blob/feat/syncBioPortal/R/cboilerplate.R#L259) shows how `nfportalutils` makes use of the mapping file.

### CODE REVIEW

File: mappings/cBioPortal.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      description: Sex of the patient
+      attribute_type: PATIENT
+      data_type: STRING
+  - source: nf1Genotype
+    target:
+      label: NF1_GENOTYPE
+      description: NF1 genotype of the patient
+      attribute_type: PATIENT
+      data_type: STRING
+  - source: nf1Genotype
```
Comment (allaway) [inline mappings/cBioPortal.yaml:24]:
```suggestion
  - source: nf2Genotype
```

**Review by allaway — APPROVED** [review]
One small suggested change, otherwise LGTM!

File: mappings/cBioPortal.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      data_type: STRING
+  - source: sex
+    target:
+      label: SEX
+      description: Sex of the patient
+      attribute_type: PATIENT
+      data_type: STRING
+  - source: nf1Genotype
+    target:
+      label: NF1_GENOTYPE
```
Comment (allaway) [inline mappings/cBioPortal.yaml:20]:
```
C002173	NF1_GERMLINE
C002174	NF1_MUTATION
C002175	NF1_SOMATIC
```

**Comment by allaway** [comment]
We've circled around this before, but we might consider a reverse mapping to oncotree code (that is, add it automatically to our annotations based on tumorType. I did this for a subset of tumors (LGG). It was not super straightforward, but it should be easier for most of our tumors (mainly because ONCOTREE has good specificity for LGG subtypes and less so for other NF tumors...) https://www.synapse.org/#!Synapse:syn27934598/tables/

File: mappings/cBioPortal.yaml (hunk 2 above)
Comment (anngvu) [reply mappings/cBioPortal.yaml:20]: I'll be meeting with MSKCC people to confirm that we can add this as well as a few other things.

**Comment by allaway** [comment]
## Curation Review — NADIA Agent

**Synapse Project:** [syn74319813](https://www.synapse.org/#!Synapse:syn74319813)
**Source:** SRA:PRJNA744233
**PMID:** [35789380](https://pubmed.ncbi.nlm.nih.gov/35789380/) — linked via BioProject elink
**DOI:** 10.1158/2159-8290.CD-21-1671

### Changes Applied

**Fourth companion dataset** to syn74319200, syn74319609, syn74319650 — all from Cancer Discovery 2022 (PMID:35789380).

**PMID and DOI added** — project was missing PMID and DOI.

**Annotation corrections:**
- `pmid` → added `35789380`
- `doi` → added `10.1158/2159-8290.CD-21-1671`
- `studyLeads` → corrected to `Amish J Patel`, `Ping Chi`
- `institutions` → `Memorial Sloan Kettering Cancer Center`
- `fundingAgency` → `NCI NIH HHS`, `FDA HHS`
- `dataType` → corrected to `geneExpression`, `epigenomics` (RNA-seq + chromatin context)
- `resourceStatus` → removed from all 108 File entities

**File-level annotations:**
- `assay`: `rnaSeq`
- `species`: `Homo sapiens`
- `tumorType`: `Malignant Peripheral Nerve Sheath Tumor`
- `platform`: `Illumina`

**Files updated:** 108

**Pull request merged**